mockall/src/lib.rs

```diff
@@ -1554,6 +1554,14 @@ impl From<RangeToInclusive<usize>> for TimesRange {
     }
 }
 
+#[derive(PartialEq)]
+#[doc(hidden)]
+pub enum ExpectedCalls {
+    Satisfied,
+    SatisfiedOnlyLowerBound,
+    SatisfiedOnlyUpperBound,
```

Comment:
I find this nomenclature confusing, because it focuses on which bounds are satisfied, rather than which aren't, which is more interesting.  I suggest
```suggestion
    Satisfied,
    TooFew,
    TooMany,
```

Reply:
Fine, will update